src/app/components/home/product-results/product-results.component.ts

```diff
@@ -0,0 +1,14 @@
+import { NgFor, NgIf } from '@angular/common';
+import { Component, Input } from '@angular/core';
+import ProductInterface from '../../../interfaces/products';
+
+@Component({
+  selector: 'app-product-results',
+  standalone: true,
+  imports: [NgIf, NgFor],
```

Comment:
Do you need these imports for the new control flow? Genuine question

Reply:
oh no it isn't, I was using the old control flow, but chnanged it to the new one, so I can remove those imports